doloop.py

```diff
@@ -665,6 +665,9 @@ def bump(dbconn, table, id_or_ids, lock_for=0, auto_add=True, test=False):
     :param str table: name of your task loop table
     :param id_or_ids: ID or list of IDs
     :param lock_for: Number of seconds that the IDs should stay locked.
+                     If a 2-tuple (min_lock_for, max_lock_for) is provided,
```

Comment:
I did not quite understand why we need to pass in min_lock_for and max_lock_for to this method. If all the ids in a particular call are going to get the same lock_for why does the called not set lock_for to the random value?